index.rst

```diff
@@ -59,6 +58,17 @@ on our `Help Desk channel on Discord`_.
 
 For sales, please use the `contact form on our website`_.
 
+.. Adding TODO list visible only in dev-build.
+   To create a dev-build locally run:
+   sphinx-build -b html . build/html -t is_dev_build
+
+.. only:: is_dev_build
+
+   To do list
+   ----------
+
+   .. todolist::
```

Comment:
I think this change is purely technical:

* The `todolist` refers to Sphinx todo directives that exist in our code. For example in [Tracking changes with ghosts and hooks](https://github.com/Certora/Documentation/blob/master/docs/user-guide/ghosts.md?plain=1#L24-L28) (this todo comment is from September) see image below.
* Before February these were shown in our Documentation. Since then they are omitted from the build, but can be recovered in a local builds using `-t is_dev_build` argument.
* The current change just adds a list of all these todo comments in the landing change, when building locally with  `-t is_dev_build` argument.

![image](https://github.com/Certora/Documentation/assets/123297287/3945b05b-a285-4a72-a4f7-bbc0d1bf056c)

Reply:
Personally I think these todo comments are superior to Jira tasks: 
1. In a todo comment I know at least which file and what lines it refers to, (as opposed to various Jira tasks that only have an obscure title).
2. Whenever anyone changes the relevant file, there is a chance they will also resolve the todo comment. As opposed to old Jira tasks that fester at the bottom of the backlog.